bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/CompactionTest.java

```diff
@@ -245,7 +245,14 @@ public void checkpointComplete(Checkpoint checkPoint, boolean compact)
         InterleavedLedgerStorage storage = new InterleavedLedgerStorage();
         storage.initialize(
             conf,
-            LedgerManagerFactory.newLedgerManagerFactory(conf, zkc).newLedgerManager(),
+            LedgerManagerFactory
+                .newLedgerManagerFactory(
+                    conf,
+                    new ZkLayoutManager(
```

Comment:
try to new layout manager from registration manager

Reply:
Thanks, will change it.